types/testing-library__jest-dom/index.d.ts

```diff
@@ -253,7 +253,54 @@ declare namespace jest {
         toHaveClass(...classNames: string[]): R;
         /**
          * @description
-         * Assert whether an element has focus or not.
+         * Check whether the given form element has the specified value.
+         *
+         * Accepts `<input>`, `<select>`, and `<textarea>` elements with the exception of `<input type="checkbox">` and
+         * `<input type="radiobox">`, which can be matched only using
+         * [toBeChecked](https:github.com/testing-library/jest-dom#tobechecked) or
+         * [toHaveFormValues](https:github.com/testing-library/jest-dom#tohaveformvalues).
+         * @example
+         * <label for="input-example">First name</label>
+         * <input type="text" id="input-example" value="Luca" />
+         *
+         * <label for="textarea-example">Description</label>
+         * <textarea id="textarea-example">An example description here.</textarea>
+         *
+         * <label for="single-select-example">Fruit</label>
+         * <select id="single-select-example">
+         *   <option value="">Select a fruit...</option>
+         *   <option value="banana">Banana</option>
+         *   <option value="ananas">Ananas</option>
+         *   <option value="avocado">Avocado</option>
+         * </select>
+         *
+         * <label for="mutiple-select-example">Fruits</label>
+         * <select id="multiple-select-example" multiple>
+         *   <option value="">Select a fruit...</option>
+         *   <option value="banana" selected>Banana</option>
+         *   <option value="ananas">Ananas</option>
+         *   <option value="avocado" selected>Avocado</option>
+         * </select>
+         *
+         * const input = screen.getByLabelText('First name')
+         * const textarea = screen.getByLabelText('Description')
+         * const selectSingle = screen.getByLabelText('Fruit')
+         * const selectMultiple = screen.getByLabelText('Fruits')
+         *
+         * expect(input).toHaveDisplayValue('Luca')
+         * expect(textarea).toHaveDisplayValue('An example description here.')
+         * expect(selectSingle).toHaveDisplayValue('Select a fruit...')
+         * expect(selectMultiple).toHaveDisplayValue(['Banana', 'Avocado'])
+         *
+         * @see
+         * [testing-library/jest-dom#tohavedisplayvalue](https:github.com/testing-library/jest-dom#tohavedisplayvalue)
+         */
+        toHaveDisplayValue(value?: string | string[]): R;
```

Comment:
nice work on those docs!

Reply:
Copy/pasted from their github docs, like the above examples :)

Reply:
It's the best way to be, my first DT PR were the exact same for Jest: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14345

Reply:
The case `toHaveDisplayValue()` without an argument does not work. I get the following error:
`TypeError: Cannot read property 'toString' of undefined`

Also see the jest-dom documentation: https://github.com/testing-library/jest-dom#tohavedisplayvalue